build.sbt

```diff
@@ -57,11 +57,12 @@ lazy val hq = (project in file("hq"))
       "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
       "com.fasterxml.jackson.module" %% "jackson-module-scala" % jacksonVersion,
       "com.google.cloud" % "google-cloud-securitycenter" % "2.20.0",
-      "org.scalatest" %% "scalatest" % "3.2.14" % Test,
+      "org.scalatest" %% "scalatest" % "3.2.15" % Test,
       "org.scalatestplus" %% "scalacheck-1-15" % "3.2.11.0" % Test,
       "org.scalacheck" %% "scalacheck" % "1.17.0" % Test,
       "com.github.alexarchambault" %% "scalacheck-shapeless_1.15" % "1.3.0" % Test,
-      "com.gu" %% "anghammarad-client" % "1.2.0",
+      "com.gu" %% "anghammarad-client" % "1.7.4",
+      "org.json" % "json" % "20230227", //evict 20180130 from anghammarad-client
```

Comment:
I'd suggest bumping this downstream instead, else we'd have a similar PR to this in Riff-Raff, AMIgo, etc, and another to undo the changes when bumping `anghammarad-client` to the next version.

Reply:
https://github.com/guardian/anghammarad/pull/91 has been released in v1.7.5 now. Just need to wait for it to [appear in Maven](https://central.sonatype.com/search?smo=true&q=anghammarad&namespace=com.gu).